enclave/core/hostcalls.c

```diff
@@ -41,32 +41,6 @@ void* oe_host_calloc(size_t nmemb, size_t size)
     return ptr;
 }
 
-void* oe_host_realloc(void* ptr, size_t size)
-{
-    oe_realloc_args_t* arg_in = NULL;
-    uint64_t arg_out = 0;
-
-    if (!(arg_in =
-              (oe_realloc_args_t*)oe_host_calloc(1, sizeof(oe_realloc_args_t))))
-        goto done;
-
-    arg_in->ptr = ptr;
-    arg_in->size = size;
-
-    if (oe_ocall(OE_OCALL_REALLOC, (uint64_t)arg_in, &arg_out) != OE_OK)
-    {
-        arg_out = 0;
-        goto done;
-    }
-
-    if (arg_out && !oe_is_outside_enclave((void*)arg_out, size))
```

Comment:
This is the check we need to incorporate in the new version.

Reply:
Got it. Thanks!